src/main/kotlin/io/easybreezy/project/model/issue/Priority.kt

```diff
@@ -0,0 +1,26 @@
+package io.easybreezy.project.model.issue
+
+import io.easybreezy.infrastructure.exposed.dao.Embeddable
+import io.easybreezy.infrastructure.exposed.dao.EmbeddableClass
+import io.easybreezy.infrastructure.exposed.dao.EmbeddableTable
+
+class Priority private constructor() : Embeddable() {
+    var value by PriorityTable.value
+
+    companion object : EmbeddableClass<Priority>(Priority::class) {
+        override fun createInstance(): Priority {
+            return Priority()
+        }
+
+        fun create(value: Int = 0): Priority {
+            val priority = Priority()
+            priority.value = value
+
+            return priority
+        }
+    }
+}
+
+object PriorityTable : EmbeddableTable() {
+    val value = integer("priority")
```

Comment:
I'm not sure about integer. Will be better to add the ability to make a priority as string-like Hight, Low

Reply:
The main purpose to have possibility raise an existing issue as a highest or lowest

Reply:
In addition to the early complexity I see "filtering" problem. How will we filter issues with high priority?

Reply:
Is that realy important in which order medium priorty tasks will be done? 
For urgent tasks status "Extremely high" can be added.

Reply:
@ lady-ann now you proposing like GitHub issues where we can move issues on top or down on our board. But I agree with @hyglok about filtering and we need to have the opportunity of highlighting high  issues

Reply:
filtering is possible: high - with priority > 0

Reply:
also now there are benefits with the easier sorting

Reply:
I won't set numbers in the filtering form. It will be magic numbers like where priority > 100 and < 10000. It's not user friendly. My proposal to save numbers of priority but add human names. Numbers add complexity to understanding. 100 it's a high priority or low?

Reply:
0 medium, <0 - low, >0 high

Reply:
Why should we use this ordering? We should add text information about priority. Because it isn't informative. Only you know about magic ordering 0 medium, <0 - low, >0 high.

Reply:
It is no magic, it is a rule. I have already wrote why.

Reply:
I can convert it for api in strings

Reply:
It's what you imagine. For me and @hyglok it's hidden and bad rules. We propose text statuses for explicit using

Reply:
> I can convert it for api in strings

how it will be? 0 as "0"?

Reply:
I need the ability to create text statuses with priority. Or maybe it will be better to rename the priority to label and add a priority there.

Reply:
how it will be? 0 as "0"?

> > I can convert it for api in strings
> 
> how it will be? 0 as "0"?

No, as I wrote "0 medium, <0 - low, >0 high"

Reply:
As I mentioned we need to avoid number with implicit behavior. I wont to see text label on priority and I can to create more than 3 priority/labels. you can’t add text for view if we have more than 3 priority. for example I need next statuses: urgent, high, normal, low, backlog.

Reply:
Fixed implicit behavior. My proposal create labels for urgent and backlog without priority

Reply:
I try to tell you that I need to have not only three types of priority. I can create 10 different priorities, or zero. I agree it’s my fault because I didn’t write task how exactly it should be. I need ability to create so many priorities as I need and I should have opportunity to set name of this priorities

Reply:
Priority is just for ordering issues I think we need add really meaningful labels. Cause what is high priority, why an issue should have even in description this word "high" when it is already on top of all issues.